shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/SpeakersDetailsScreen.kt

```diff
@@ -46,7 +46,7 @@ fun SpeakersFlow(
     }
 
     Column(Modifier.fillMaxWidth()) {
-        NavigationBar(title = "SPEAKERS", isLeftVisible = true, onLeftClick = {
+        NavigationBar(title = "Speakers", isLeftVisible = true, onLeftClick = {
```

Comment:
Don't you want to extract it to the strings resources?

Reply:
fixed